### PR TITLE
Jmrunge/melted status

### PR DESCRIPTION
**Pull request by jmrunge**
Adds fps, currentFrame and totalFrames to status from mvcp driver.

### CODE REVIEW

File: test/mosto-status-test.js [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-        };
 
 //TODO: This test should be rewritten after @fabriciocosta merges his part with more usefull data!
 describe('Mosto status', function() {
+    var self = this;
+    self.rec = 0;
+    self.mosto_status = undefined;
     before(function(done) {
-        server = new mosto(config);
+        server = new mosto();
```
Comment (Lacrymology) [inline test/mosto-status-test.js:12]: why did you remove the config?

File: test/mosto-status-test.js [hunk 2]
```diff
@@ -31,14 +27,34 @@ describe('Mosto status', function() {
     });
 
     describe('suscribe to status and wait 5 seconds', function() {
-        it('--should have received 5 status events', function() {
+        before(function(done) {
+            var id = setInterval(function() {
```
Comment (Lacrymology) [inline test/mosto-status-test.js:31]: This doesn't make much sense. If the status emitter fails here, the `before all` will fail instead of the `5 status events` test.

File: test/mosto-status-test.js (hunk 1 above)
Comment (jmrunge) [inline test/mosto-status-test.js:12]: If you read at messages commit you will see what I told about this: config is pointless because this was just for using json driver and get a playlist up, but json driver is broken, it cant be use until the methods added to mongo driver (and used by mosto!) are added to it.  :)

File: test/mosto-status-test.js (hunk 2 above)
Comment (jmrunge) [inline test/mosto-status-test.js:31]: You're right.  I moved this to before because it wasnt working and finally it was something else and forgot to put it back inside it.  Will commit a change to this right now.

**Comment by jmrunge** [comment]
Please review after my next commit.  As for the test as it was before... Tell me why it was passing even when the status emit event was NEVER CALLED FROM INSIDE MOSTO (as @fabriciocosta and you removed the setInterval that emitted it)

**Comment by Lacrymology** [comment]
hm, let me check the code

**Comment by Lacrymology** [comment]
it was because the 

```
it('--should have received 5 status events', function() {
```

was missing the `done` parameter

``` patch
-    it('--should have received 5 status events', function() {
+    it('--should have received 5 status events', function(done) {
```

**Comment by jmrunge** [comment]
I added the done and it still passed... You were never doing some kind of
assert, should or whatever inside the it.  Please take a look at my last
commit to see if now you like the test...

2013/4/23 Tomas Neme notifications@github.com
